packages/polyfill-library/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "polyfill-library",
-  "version": "3.25.1",
+  "version": "0.0.0",
```

Comment:
These are being reset because they were never published?

Reply:
They were reset because the publishing workflow was going to use https://github.com/matthew-andrews/npm-prepublish#npm-prepublish. We have since moved away from this and we now use this -- https://github.com/Financial-Times/polyfill-service/tree/master/packages/polyfill-service#release-process